### PR TITLE
Give IIIF Manifests Some Wings

### LINKED ISSUE #5474: Image viewer is not working in nurax-pg
### Prerequisite

Issue #5480 should be resolved before exploring this issue.  There are several issues with files on a work show page.  When they are resolved, it is possible the viewer will work as expected.

### Descriptive summary

Cannot view images using image viewer, e.g. IIIF viewer.

### Steps to reproduce the behavior

1. Create work of image type and check if images can be viewed using image viewer.

### Related Work

Issue #5480 - image file in Items section of work show page is a link instead of a title
Issue #4788 - Work thumbnail fails to show in Dashboard -> Works list of works

### PR DESCRIPTION
**Pull request by orangewolf**
### Fixes

Fixes #5474 , #5543 

### Summary

Adds needed search parameters and RIIIF settings to get IIIF manifests and UV images to display properly when using Valkyrie.

### Guidance for testing, such as acceptance criteria or new user interface behaviors:
* Add a work with one or more images
* Wait for derivatives to be generated
* View that the UV shows the uploaded images on the work show page
* add '/manifest.json' to the work show page URL to see the IIIF manifest

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: app/indexers/hyrax/valkyrie_file_set_indexer.rb [hunk 1]
```diff
@@ -99,6 +102,11 @@ def to_solr # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Met
 
     private
 
+    # Convert Valkarie Original File Pointer to versioned url sysntax expected by the iiif_presenter
```
Comment (alishaevn) [inline app/indexers/hyrax/valkyrie_file_set_indexer.rb:105]:
```suggestion
    # Convert Valkyrie Original File Pointer to versioned url syntax expected by the iiif_presenter
```

File: app/presenters/hyrax/iiif_manifest_presenter.rb [hunk 2]
```diff
@@ -93,12 +93,8 @@ def manifest_url
...
     # @return [Array<#to_s>]
     def member_ids
-      case model
-      when Valkyrie::Resource
-        Array(model.try(:member_ids))
-      else
-        Hyrax::SolrDocument::OrderedMembers.decorate(model).ordered_member_ids
-      end
+      m = model.is_a?(::SolrDocument) ? model.hydra_model : model
+      m.class < Hyrax::Resource ? Array(model.member_ids) : Hyrax::SolrDocument::OrderedMembers.decorate(model).ordered_member_ids
```
Comment (alishaevn) [inline app/presenters/hyrax/iiif_manifest_presenter.rb:97]: what is `m.class < Hyrax::Resource` doing?

Comment (dlpierce) [reply app/presenters/hyrax/iiif_manifest_presenter.rb:97]: Checks if the model's class is a subclass of Hyrax::Resource. [documentation](https://devdocs.io/ruby~3.2/module#method-i-3C)

**Review by dlpierce — APPROVED** [review]
Looks good and less work than I expected!

**Comment by orangewolf** [comment]
> Looks good and less work than I expected!

the investigation was deep but the answer was simple =-)

**Pull request merged**